### ISSUE TITLE
Problem: Makefile is building old tx-query instead of tx-query-next

### ISSUE THREAD

**Issue by allthatjazzleo** [crypto-com/chain#1810]
```
# build the enclave queury binary
build-sgx-query:
	@echo "\033[32mcompile sgx query\033[0m"; \
	docker run -i --rm \
		-v ${HOME}/.cargo/git:/root/.cargo/git \
		-v ${HOME}/.cargo/registry:/root/.cargo/registry \
		-v `pwd`:/chain \
		--env SGX_MODE=$(SGX_MODE) \
		--env RUSTFLAGS=-Ctarget-feature=+aes,+sse2,+sse4.1,+ssse3 \
		--workdir=/chain \
		$(IMAGE_RUST):latest \
		bash -c ". /root/.docker_bashrc && \
		$(CARGO_BUILD_CMD) -p tx-query-app && \
		make -C chain-tx-enclave/tx-query $(SGX_ARGS)"
```

it needs some new middlewares like ra-sp-server and some signature files need to add into the Makefile,

**Comment by leejw51crypto** [comment]
yes, old tx-query is longer used

**Comment by leejw51crypto** [comment]
ftxsgx-elf2sgxs ./tx-query2-enclave-app   --heap-size 0x2000000 --stack-size 0x80000 --threads 6  --debug

after compiling, make sure that `--debug` option attached, otherwise
strange error occurs

**Comment by allthatjazzleo** [comment]
fixed in #1844
